homeassistant/components/assist_satellite/strings.json

```
@@ -24,8 +24,12 @@
           "name": "Media ID",
           "description": "The media ID to announce instead of using text-to-speech."
         },
+        "preannounce": {
+          "name": "Preannounce",
+          "description": "Play a sound before the announcement."
+        },
         "preannounce_media_id": {
-          "name": "Preannounce Media ID",
+          "name": "Preannounce media ID",
```

In the frontend meeting we quickly talked about making this maybe this:

```suggestion
          "name": "Custom preannounce media ID",
```

Or, alternatively, adjust the description to include if it isn't set a default bell sound would be used.

Just to make it more clear, it is about adjusting existing sounds.

/CC @jlpouffier

---

Yeah either adding the word "custom" or adding something that explain that not using this field is perfectly ok and will use the default sound. 

I think adding "custom" is the simplest way to achieve that without bloating the description

---

I changed it to "Custom media ID to play before ..."